strawberry/subscriptions/protocols/graphql_ws/handlers.py

```diff
@@ -114,14 +114,14 @@ async def handle_start(self, message: OperationMessage) -> None:
             )
         except GraphQLError as error:
             error_payload = format_graphql_error(error)
-            await self.send_message(GQL_ERROR, operation_id, error_payload)
+            await self.send_message(GQL_ERROR, operation_id, error_payload)  # type: ignore # noqa: E501
```

Comment:
@DoctorJohn can you take a look at these ignores? :)

Reply:
Just took a look at it locally, we don't need those ignores. mypy and pytest agree :)

Reply:
uhm, I do get an error when removing it: 

```
strawberry/subscriptions/protocols/graphql_ws/handlers.py:117: error: Argument 3 to "send_message" of "BaseGraphQLWSHandler" has incompatible type "GraphQLFormattedError"; expected "Union[Dict[str, Any], Dict[str, Any], StartPayload, DataPayload, ErrorPayload, None]"  [arg-type]
```

Reply:
Lemme take another look 👀

Reply:
@patrick91 Silly mistake by me. Just fixed it and pushed to this branch (hope thats okay). I should take some time and refactor the legacy protocols typings. For the new protocol I used dataclasses instead of typed dicts which worked really well.

Reply:
no worries! and thanks for fixing it! 😊 might be worth open an issue for the refactoring, so we know that it will happen at some point 😊